bulbs/special_coverage/static/special_coverage/stylesheets/flexbox.scss

```diff
@@ -0,0 +1,419 @@
+//////////////////////////
```

Comment:
Should we be shifting to have a single flexbox mixin pulled in somewhere more globally?  I feel like I've seen this kind of flexbox file replicated all over but maybe I'm crazy